### ISSUE TITLE
Product Query [3]: [Variation] `Products by Category` block

### ISSUE THREAD

**Issue by sunyatasattva** [woocommerce/woocommerce-gutenberg-products-block#7320]
Implement a “Products by Category” variation of the “Product Query” block. This variation should show at least the following settings:

* Category
* Stock status
* Order by

This is blocked by https://github.com/woocommerce/woocommerce-blocks/issues/7218

**Comment by sunyatasattva** [comment]
Closing as won't do as of this comment: pdnLyh-2jZ-p2#comment-1685
